subworkflows/nf-core/vcf_impute_beagle5/main.nf

```diff
@@ -0,0 +1,101 @@
+include { BEAGLE5_BEAGLE                     } from '../../../modules/nf-core/beagle5/beagle'
+include { BCFTOOLS_VIEW                      } from '../../../modules/nf-core/bcftools/view'
+include { GLIMPSE2_LIGATE                    } from '../../../modules/nf-core/glimpse2/ligate'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_1 } from '../../../modules/nf-core/bcftools/index'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_2 } from '../../../modules/nf-core/bcftools/index'
+
+workflow VCF_IMPUTE_BEAGLE5 {
+
+    take:
+    ch_input  // channel (mandatory): [ [id], vcf, tbi ]
+    ch_panel  // channel (mandatory): [ [panel, chr], vcf, tbi ]
+    ch_chunks // channel (optional) : [ [panel, chr], regionout ]
+    ch_map    // channel (optional) : [ [chr], map]
+
+    main:
+    ch_versions = channel.empty()
+
+    // Branch input files based on format
+    ch_input
+        .branch { _meta, vcf, _tbi ->
+            bcf: vcf.toString().contains('.bcf')
+            vcf: vcf.toString().contains('.vcf')
+            other: true
+        }
+        .set { ch_input_branched }
+
+    ch_input_branched.other.map{
+        error "ERROR: ch_input files must be in VCF or BCF format."
```

Comment:
Can you also add the wrong filename here?

Reply:
Done